pslogin/src/main/scala/WorldSessionActor.scala

```diff
@@ -2880,7 +2880,7 @@ class WorldSessionActor extends Actor with MDCContextAware {
               accessedContainer = None
             }
           case Some(container) => //just in case
-            if(vel.isDefined || Vector3.DistanceSquared(player.Position, container.Position) > 25) {
```

Comment:
Are lockers the only thing affected by this? With PR #236 we can set the position of lockers (which we have now) that would render this point moot, and would be a good idea to enforce server side anyway.

Reply:
If that's the case, the distance check should be performed in the `UseItemMessage` block for Lockers and not the periodic block for `PSUM`.  The real reason the distance was to be enforced for vehicle is that vehicles can move.  Lockers and corpses can not.  The player can move too at the moment but probably shouldn't be able to move.  Whether that gets changed or not, a vehicle will still be able to move and, without a range check, its trunk would remain open on someone's screen.